wcivf/apps/people/views.py

```diff
@@ -66,9 +67,37 @@ def get_object(self, queryset=None):
         obj.title = self.get_title(obj)
         obj.intro = self.get_intro(obj)
         obj.text_intro = strip_tags(obj.intro)
-
+        obj.post_country = self.get_post_country(obj)
+        if obj.personpost and\
+                (obj.personpost.election.name == '2017 General Election'):
+            # We only care about the UK manifesto, and the candidate's own
+            # country manifesto. Return the UK manifesto first.
+            obj.manifestos_2017 = Manifesto.objects.filter(
+                party=obj.personpost.party,
+                election=obj.personpost.election).filter(
+                Q(country='UK') | Q(country=obj.post_country))
+            obj.manifestos_2017 = sorted(
+                obj.manifestos_2017,
+                key=lambda n: n.country != 'UK')
         return obj
 
+    def get_post_country(self, person):
```

Comment:
As you say, this is a bit of a hack, and fine for now (it works!), but I think I'd rather this were moved on to the `Post` model directly, in future. This can wait for a while, as we store the `territory_code` in [EveryElection](https://elections.democracyclub.org.uk/) and eventually this should be imported directly on to the Post model.